soperator/installations/example/variables.tf

```diff
@@ -274,6 +274,25 @@ variable "slurm_operator_version" {
   nullable    = false
 }
 
+# region PartitionConfiguration
+
+variable "slurm_partition_config_type" {
+  description = "Variable provide possibility for configuring custom slurm partitions"
+  default     = "default"
+  type = string
+
+  validation {
+    condition     = (contains(["default", "custom"], var.slurm_partition_config_type))
+    error_message = "Invalid partition config type. It must be one of `default` or `custom`."
+  }
+}
+
+variable "slurm_partition_raw_config" {
+  description = "Configuration as list strings started with PartitionName"
```

Comment:
```suggestion
  description = "Partition config in case of `custom` slurm_partition_config_type. Each string must be started with `PartitionName`."
```